service/llmService.js

```
@@ -312,6 +351,23 @@ class LLMService extends EventEmitter {
                     break;
 
                 case "response.text.done":
+                    // 누적된 텍스트를 대화 내역에 추가
+                    const metaDone = this.meta.get(sessionId) || {};
+                    if (metaDone.accumulatedText) {
+                        if (!this.conversations.has(sessionId)) {
+                            this.conversations.set(sessionId, []);
+                        }
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)